### PR TITLE
[ko] Update outdated in dev-1.21-ko.8 (M1-M5)

### PR DESCRIPTION
**Pull request by seokho-son**
Ref #29253
Task: M1 - M5

 M1. content/en/community/_index.html | 1(+XS) 1(-)
 M2. content/en/docs/concepts/configuration/manage-resources-containers.md | 1(+XS) 1(-)
 M3. content/en/docs/concepts/configuration/secret.md | 32(+M) 23(-)
 M4. content/en/docs/concepts/containers/images.md | 2(+XS) 1(-)
 M5. content/en/docs/concepts/containers/runtime-class.md | 1(+XS) 1(-)

### CODE REVIEW

File: content/ko/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -12,26 +12,33 @@ weight: 30
...
+쿠버네티스 시크릿은 기본적으로 API 서버의 기본 데이터 저장소(etcd)에 암호화되지 않은 상태로 저장된다. API 접근(access) 권한이 있는 모든 사용자 또는 etcd에 접근할 수 있는 모든 사용자는 시크릿을 조회하거나 수정할 수 있다.
+또한 네임스페이스에서 파드를 생성할 권한이 있는 사람은 누구나 해당 접근을 사용하여 해당 네임스페이스의 모든 시크릿을 읽을 수 있다. 여기에는 디플로이먼트 생성 기능과 같은 간접 접근이 포함된다. 
+
+시크릿을 안전하게 사용하려면 최소한 다음의 단계를 따르는 것이 좋다.
 
 1. 시크릿에 대한 [암호화 활성화](/docs/tasks/administer-cluster/encrypt-data/).
-2. 시크릿 읽기 및 쓰기를 제한하는 [RBAC 규칙 활성화 또는 구성](/ko/docs/reference/access-authn-authz/authorization/). 파드를 만들 권한이 있는 모든 사용자는 시크릿을 암묵적으로 얻을 수 있다.
+2. 시크릿의 데이터 읽기 및 쓰기(간접적인 방식 포함)를 제한하는 [RBAC 규칙](/ko/docs/reference/access-authn-authz/authorization/)
+   활성화 또는 구성.
+3. 적절한 경우, RBAC와 같은 메커니즘을 사용하여 새로운 시크릿을 생성하거나 기존 시크릿을 대체할 수 있는 주체(principal)들을 제한한다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/configuration/secret.md:40]:
```suggestion
3. 적절한 경우, RBAC과 같은 메커니즘을 사용하여 새로운 시크릿을 생성하거나 기존 시크릿을 대체할 수 있는 주체(principal)들을 제한한다.
```

- RBAC는 '알백'과 같이 발음하여 조사를 수정했습니다.
- https://youglish.com/pronounce/rbac/english?

**Review by pjhwa — COMMENTED** [review]
@seokho-son 감사합니다!

의견드립니다~

**Comment by seokho-son** [comment]
@pjhwa 리뷰 의견 반영하여 PR 업데이트 하였습니다! 간단한 사항이라 스쿼싱까지 처리하였습니다.
https://github.com/kubernetes/website/pull/30415/files#diff-c1609527b766d7a47d05fbdca289b6e393ee70ec013a6ad3d833091159ac6d63R40

**Pull request merged**